supervision/detection/vlm.py

```diff
@@ -3,7 +3,7 @@
 import json
 import re
 from enum import Enum
-from typing import Any, Dict, List, Optional, Tuple, Union
+from typing import Any, Optional, Union
```

Comment:
lets replace `Union` with `|` and `Optional` with `| None` to keep things consistent